src/cbapi/psc/livequery/models/facet.yaml

```diff
@@ -0,0 +1,22 @@
+type: object
+required: [] # TODO(ww): Find out which result fields are required
+properties:
```

Comment:
Just to state my assumption the reason you didn't include the terms property is because this is a singular facet object which is inside of terms right?

https://developer.carbonblack.com/reference/psc/cb-liveops/latest/livequery-api/#filters-on-results

Reply:
You don't need terms to run the query successfully. You won't actually get anything back, but the call won't fail.